.github/workflows/tests.yml

```
@@ -41,6 +41,9 @@ jobs:
     - name: Run pytest
       run: >
         pytest --durations=10 --cov=sphinx_book_theme --cov-report=xml --cov-report=term-missing
+    - name: Run parallel build test
```

```suggestion
    # Running the parallel build test on its own because we cannot pass parallel config to Sphinx via pytest
    - name: Run parallel build test
```

does this correctly encapsulate why we're doing this outside of pytest?

---

I will make the comment more explicit about the need to pass parallel=2 to the sphinx constructor, including the [link](https://github.com/sphinx-doc/sphinx/blob/d8c006f1c0e612d0dc595ae463b8e4c3ebee5ca4/sphinx/testing/util.py#L101)
I also will move the test into pytest as I did for this other extension:
https://github.com/readthedocs/sphinx-notfound-page/blob/47887ecb35f42da29dd111f8a281a22ad4bf7412/tests/test_urls.py#L33

---

err, maybe. I specifically moved away from running sphinx using subprocess in pytest, because it is harder to debug and breaks in Windows ae76e1286d6378da296253dce0ce40305d8c5197

---

I moved it before I saw your reply. I can put it back to the yml if you prefer. I will also try to add support to SpinxTestApp, but it might take a long time before it shows up in a release.

---

Nah it's ok I just wanted to point this out

---

OK. The SphinxTestApp PR is https://github.com/rscohn2/spion/sphinx/pull/8257